source/features/fix-view-file-link-in-pr.tsx

```diff
@@ -27,12 +28,13 @@ function handleMenuOpening(event: delegate.Event): void {
 	// - Branches with slashes in it
 	// - PRs opened from the default branch
 	const headReferenceLink = select<HTMLAnchorElement>('.head-ref a')!;
-	const [, owner, repository] = headReferenceLink.pathname.split('/', 3); // Example pathname: '/kidonng/refined-github/tree/fix-console-error'
-	const branch = headReferenceLink.title.replace(/^[^:]+:/, ''); // Example title: 'tejanium/refined-github:bra/nch' or just 'local-branch`
+	const pathParts = parseRoute(headReferenceLink.pathname);
+	pathParts[3] = 'blob';
```

Comment:
I wish I could do this
```suggestion
	const viewFile = select<HTMLAnchorElement>('[data-ga-click^="View file"]', dropdown)!;
	const pathParts = parseRoute(viewFile.pathname);
	pathParts[3] = 'blob';
	viewFile.pathname = pathParts.join('/');
```

But it will error out since there is no branch there just a commit
Example /yakov116/refined-github/blob/4b08f2421fafc39b19214c0b9a9ecb4d04d824c3/source/features/conflict-marker.tsx